src/webview/main-refactored.ts

```
@@ -0,0 +1,563 @@
+
+    console.log('📱 [MANAGER] Event listeners set up');
+  }
+
+  /**
+   * メッセージ処理
+   */
+  private async handleMessage(message: WebviewMessage): Promise<void> {
+    try {
+      switch (message.command) {
+        case TERMINAL_CONSTANTS.COMMANDS.INIT:
+          await this.handleInitMessage(message as any);
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)